usage.md

```diff
@@ -0,0 +1,10 @@
+# Usage Stats Collection
+
+As of v0.1.9, Aqueduct collects usage statistics, which helps us identify bugs and better understand
+how to improve our user experience. After starting the server, a prompt will show up informing that
```

Comment:
Nit: It's not a prompt, right? We're not asking them to interact with it in anyway. 

I would just say the server start logs will notify you that usage information is being collected.